src/main/java/org/openhim/mediator/normalization/ParseProvideAndRegisterRequestActor.java

```diff
@@ -38,12 +52,24 @@ public static ProvideAndRegisterDocumentSetRequestType parseRequest(String docum
 
 
     private void processMsg(SimpleMediatorRequest<String> msg) {
+        ActorRef requestHandler = msg.getRequestHandler();
         try {
+            CoreResponse.Orchestration orch = new CoreResponse.Orchestration();
+            orch.setName("Parse Provider and Register Document Set.b contents");
+            orch.setRequest(new CoreResponse.Request());
```

Comment:
I wonder if we should not construct this orch unless the 'sendParseOrchestration' property is set.